docs/source/user_guide/runtime-image-conf.md

```diff
@@ -118,6 +118,24 @@ $ elyra-metadata install runtime-images --schema_name=runtime-image \
        --replace
 ```
 
+#### Exporting runtime image configurations
+
+To export runtime image configurations:
+
+```bash
+elyra-metadata export runtime-images \
+	--schema-name="runtime-image" \
+	--directory="/tmp/foo"
+```
+
+The above example will export all runtime image configurations in the "runtime-image" schema to the "/tmp/foo/runtime-images" directory.
+
+Note that you must specify the `--directory` option. If the `--schema-name` option is not specified, runtime image configurations across all schemas will be exported. 
```

Comment:
```suggestion
Note that you must specify the `--directory` option. If the `--schema_name` option is not specified, runtime image configurations across all schemas will be exported. 
```

Reply:
See above (remove `--sche...` reference to keep it simple.

Reply:
Removed all mentions of `--schema-name` parameter from the documentation as discussed via chat.